samples/sensor/scd30/src/main.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021 Advanced Climate Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <stdio.h>
+
+void main(void)
+{
+	const struct device *dev = device_get_binding("SCD30_0");
```

Comment:
Please use DEVICE_DT_GET_ANY or DEVICE_DT_GET_ONE. Make sure to check for readiness with device_is_ready.